worlds/archipidle/Rules.py

```diff
@@ -4,36 +4,33 @@
 
 
 class ArchipIDLELogic(LogicMixin):
-    def _archipidle_location_is_accessible(self, player_id, items_required):
-        items_received = 0
-        for item in self.prog_items:
-            if item[1] == player_id:
-                items_received += 1
-
-        return items_received >= items_required
+    def _archipidle_location_is_accessible(self, player_id, items_required, world):
```

Comment:
self.world is the same as the world passed here, so this parameter is not needed. Also both are multiworld

Reply:
Oh wow this is really old code if I didn't notice that. Probably going to do a pass over this world in the morning